tools/utils/download_blender.sh

```diff
@@ -352,27 +254,27 @@ if [ "${version}" = "all" ]; then
     wait_for_all "${pids[@]}"
 else
     if [ "${os}" == "Mac" ]; then
-        ver=v${version}
-        url=${BLENDER_DOWNLOAD_URL_MACOSX[${ver}]}
+        ver=${version}
+        url=${BLENDER_DOWNLOAD_URL_MACOSX["${ver}"]}
         move_from=""
-        if [[ "${NEED_MOVE_MACOSX[${ver}]+_}" == "_" ]]; then
-            move_from=${NEED_MOVE_MACOSX[${ver}]}
+        if [[ "${NEED_MOVE_MACOSX["${ver}"]+_}" == "_" ]]; then
```

Comment:
I want to check if this works correctly.
Especially, does `"${NEED_MOVE_MACOSX["${ver}"]+_}"` work correctly?